weatherApp/Constants/Constants.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Constants.swift
+//  weatherApp
+//
+//  Created by Luyanda Sikithi on 2022/11/04.
+//
+
+import Foundation
+struct Constants {
```

Comment:
You can change this struct into an enum. You keep the static let declarations and the way you call it is exactly the same, but there is no chance of creating an instance. It basically makes the intention of the class clear, that it is used for namesapce.